Examples/Tests/point_of_contact_EB/inputs_3d

```diff
@@ -12,12 +12,6 @@ geometry.prob_hi     = 0.26 0.26 0.26
 
 boundary.field_lo = pec pec pec
 boundary.field_hi = pec pec pec
```

Comment:
I removed these lines in order to avoid triggering the electrostatic solver, for this particular test.